hack/errata.py

```diff
@@ -38,6 +39,7 @@ def run(poll_period=datetime.timedelta(seconds=3600), cache=None, webhook=None,
             if cache and message['fulladvisory'] in cache or 'bug fix update' not in message['synopsis']:
                 continue
             notify(message=message, webhook=webhook)
+            lgtm_fast_pr_for_errata(message)
```

Comment:
Can we shift this up, return any approved PR, and mention it in the `notify` message?

Reply:
Included URL in the notification - 81af18437430ec02fb94bc314ae57c0f803d3c83